utils/src/main/scala/za/co/absa/enceladus/utils/general/JsonUtils.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.general
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object JsonUtils {
+
+  def prettyJSON(jsonIn: String): String = {
+    val mapper = new ObjectMapper()
+
+    val jsonUnindented = mapper.readValue(jsonIn, classOf[Any])
+    val indented = mapper.writerWithDefaultPrettyPrinter.writeValueAsString(jsonUnindented)
+    indented.replace("\r\n", "\n")
+  }
+
+  def prettySparkJSON(jsonIn: String): String = {
+    val properJson = "[" + "}\n".r.replaceAllIn(jsonIn, "},\n") + "]"
+    prettyJSON(properJson)
+
+  }
+
+  def getDataFrameFromJson(spark: SparkSession, json: String): DataFrame = {
```

Comment:
isn't sparkSession usually passed as implicit? Could it be even here?

Reply:
Yes, this test suite inherits SparkTestBase which creates a special instance of SPark session in local mode.